pysqream_blue/cursor.py

```diff
@@ -30,15 +30,26 @@ def __init__(self, context_id, query_timeout, call_credentialds, use_ssl,
         self.host = host
         self.port = port
         self.options = options
-        self.channel = grpc.secure_channel(f'{self.host}:{self.port}', grpc.ssl_channel_credentials(),
-                                           options=self.options)
-        self.client = qh_services.QueryHandlerServiceStub(self.channel)
+        self._create_channel()
+        self._create_query_handler_service_stub(self.channel)
 
         if len(self.logs.logger.handlers) == 0 and self.logs.start:
             self.logs.set_log_path(log_path=log_path)
             self.logs.set_level(log_level)
             self.logs.start_logging(__name__)
 
+    def _restart_channel_and_client(self):
+        self.channel.close()
```

Comment:
maybe it's better to create a function `_close_channel` and do `self.channel.close()` only on the function, and any places that we doing `self.channel.close()` we can change to `_close_channel`.